pkg/executor/api.go

```diff
@@ -44,14 +43,12 @@ func (executor *Executor) getServiceForFunctionAPI(w http.ResponseWriter, r *htt
 	}
 
 	// get function metadata
-	m := metav1.ObjectMeta{}
-	err = json.Unmarshal(body, &m)
+	fn := &fv1.Function{}
+	err = json.Unmarshal(body, &fn)
 	if err != nil {
 		http.Error(w, "Failed to parse request", http.StatusBadRequest)
 		return
 	}
-
-	fn, err := executor.fissionClient.CoreV1().Functions(m.Namespace).Get(m.Name, metav1.GetOptions{})
 	if err != nil {
```

Comment:
Error handling was done for the API call

Reply:
We also need to fill in the function struct with proper values, right?

Reply:
Done